High level introduction to ASDF.ipynb

```diff
@@ -0,0 +1,210 @@
+    "- Ability to have different attributes reference the same data or object without making copies, so that data or object can be shared by many things.\n",
+    "- Full support for all standard numerical binary values.\n",
+    "- Supports mode of putting some or all data \"in-line\", i.e., as text values.\n",
+    "- Supports very flexible WCS model.\n",
+    "- Mechanism for extensions to the standard format, both in the standard itself, and for local purposes.\n",
+    "- Versioned globally, and for specific extensions.\n",
+    "- Support for schema files that can be used to validate the proper structure of the file, both for reading and writing.\n",
+    "- Support for streamed data.\n",
+    "- Language agnostic, but currently only fully supported in Python\n",
+    "\n",
+    "Many of these items value may not yet be obvious to the reader, but tutorials will illustrate  the value with examples."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## FITS Features not yet Supported"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Variable length records in tables (using heap storage)\n",
+    "- Random Groups"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisites to Tutorials\n",
+    "\n",
+    "### Installations\n",
+    "\n",
+    "Of course Python 3.6 or later must be installed, along with ipython, jupyter, numpy, astropy, gwcs, and asdf. We generally recommend installing Python using miniconda since conda allows easy cloning of different Python environments ([miniconda download](https://docs.conda.io/en/latest/miniconda.html) and using pip install for the rest after activating the miniconda environment. For example, at the shell level:\n",
+    "\n",
+    "`\n",
+    "pip install ipython\n",
+    "pip install jupyter\n",
+    "pip install numpy\n",
+    "pip install astropy\n",
+    "pip install asdf\n",
+    "pip install gwcs\n",
+    "`\n",
+    "### Data Files\n",
+    "\n",
+    "The tutorials generally require accessing sample ASDF files. The tutorials usually do this by referring to files on the network that will be downloaded if not already present on your system. The needed downloads will be part of the tutorials."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Tutorials"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To be added"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## References and Documentation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- [Original ASDF Paper](https://www.sciencedirect.com/science/article/pii/S2213133715000645)\n",
+    "- [Python ASDF Library Documenation](https://asdf.readthedocs.io/en/2.7.1/)\n",
```

Comment:
Recommend changing this link to the stable docs version:

https://asdf.readthedocs.io/en/stable